runtime/doc/options.txt

```diff
@@ -2139,7 +2139,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 'endofline' 'eol'	boolean	(default on)
 			local to buffer
 	When writing a file and this option is off and the 'binary' option
-	is on, or 'fixeol' option is off, no <EOL> will be written for the
+	is on, or 'fixeol' option is off, no <EOL> will be written at the
```

Comment:
keep

Reply:
Why?

Reply:
Because Vim has not changed this.

Reply:
See https://github.com/neovim/neovim/pull/20855/files

Reply:
Sorry, I don't see the point. There was a typo (`for at`), which we fixed one way at some point and Bram fixed another way. We tend to follow upstream unless there are pressing reasons, so I switched the fix.

Reply:
Why is there a typo?

Reply:
Oh, I see what you mean, I was looking at the wrong section.

(Using more than a single word for comments may safe some tedious -- for both parties -- back and forth.)

Reply:
The current docs is exactly the same as upstream, and you are diverging from upstream by changing this.